pkg/ebpf/events_capabilities.go

```diff
@@ -0,0 +1,37 @@
+package ebpf
+
+import (
+	"github.com/syndtr/gocapability/capability"
+)
+
+func GetCapabilitiesRequiredByEvents(events []int32) []capability.Cap {
+	reqCapabilities := make(map[capability.Cap]bool)
+	for _, e := range events {
+		addEventAndDependenciesCapabilities(e, reqCapabilities)
+	}
+
+	capList := make([]capability.Cap, len(reqCapabilities))
+	i := 0
+	for reqCap := range reqCapabilities {
+		capList[i] = reqCap
+		i++
```

Comment:
append

Reply:
Again, I use make with defined length to reduce runtime (so I need to address the index in the defined array).